addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -338,7 +338,33 @@ export class WebsitePreview extends Component {
         this.title.setParts({ action: currentTitle });
     }
 
-    _onPageLoaded() {
+    _onPageLoaded(ev) {
+        // FIX Chrome-only. If you have the backend in a language A but the
+        // website in English only, you can 1) modify a record's (event,
+        // product...) name in language A (say "New Name").
+        // 2) visit the page `/new-name-11` => the server will redirect you to
+        // the English page `/origin-11`, which is the only one existing.
+        // Chrome caches the redirection.
+        // 3) give the same name in English as in language A, try to visit
+        // => the server now wants to access `/new-name-11`
+        // => Chrome uses the cache to redirect `/new-name-11` to `/origin-11`,
+        // => the server tries to redirect to `/new-name-11` => start again 🔄.
+        // Chrome injects a "Too many redirects" layout in the iframe, which in
+        // turn raises a CORS error when the app tries to update the iframe.
+        // If we detect that behavior, we reload the iframe with a new query
+        // parameter, so that it's not cached for Chrome.
+        if (navigator.userAgent.toLowerCase().includes("chrome") && !this.iframeReloaded) {
```

Comment:
You don't really need this new `iframeReloaded` property, you could just do 
```suggestion
        if (navigator.userAgent.toLowerCase().includes("chrome") && !this.iframe.el.src.includes('iframe_reload')) {
```
As the src is still the one requested, despite chrome adding its own layout?